easydita-knowledge-portal/functions.php

```diff
@@ -128,7 +128,7 @@ function easydita_knowledge_portal_scripts() {
 function easydita_knowledge_portal_change_style_version_num($styles) {
 	$styles -> default_version = "123";
 }
-//add_action("wp_default_styles", "easydita_knowledge_portal_change_style_version_num");
+add_action("wp_default_styles", "easydita_knowledge_portal_change_style_version_num");
```

Comment:
This shouldn't get committed, by default we want the basic caching that WP uses for the style.css file.